### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+FROM debian:bullseye as builder
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Is it possible to add a package stage here?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by xansec — APPROVED** [review]
LGTM!

**Comment by xansec** [comment]
Hmm, this seems to be failing
https://github.com/mayhemheroes/freeswitch/actions/runs/3839240997

**Comment by rnshah9** [comment]
#2 to fix.

**Pull request merged**